test/acceptance/README.md

```diff
@@ -1,10 +1,230 @@
+
+### Run acceptance tests with remote instance of SBO
+
+It is possible to run the acceptance tests with SBO already running, that is without building and starting the local instance and without installing the dependent resources (such as CRD, role, role-binding, service-account, etc.) required when the local instance is used. That is particulary useful for testing SBO installed on a remote cluster or running the acceptance tests with various versions of the SBO.
+
+For that there is `TEST_ACCEPTANCE_START_SBO` environment variable that needs to be set to `remote`.
+
+The remote cluster where the SBO should be running is determined by the `KUBECONFIG`.
+
+To run acceptance tests with remote instance of SBO:
+
+```bash
+TEST_ACCEPTANCE_START_SBO=remote make test-acceptance
+```
+
+Compared with the defaults this command only:
+* Deletes the old namespace from any previous run,
+* Creates a new namespace with generated name like `test-namespace-xxxxxxxx`,
+* Executes the acceptance tests.
+
+The `remote` option requires to specify the namespace where the operator is running by setting the `SBO_NAMESPACE` env variable.
+
+To run acceptance tests with remote instance of SBO running in a given namespace (e.g. `sbo`):
+
+```bash
+TEST_ACCEPTANCE_START_SBO=remote SBO_NAMESPACE=sbo make test-acceptance
+```
+
+
+### Run a sub-set of features or scenarios
+
+It is possible to run a sub-set of features or even a single scenario (for example, when you are working on a new scenario and you need to run it over and over). The way how to do it is by "marking" that feature or scenario by a "tag" - directly in the particular `*.feature` file - in a form of `@<tag>` (e.g. `@wip`). Something like the following:
+
+```gherkin
+Feature: ...
+
+   @wip
+   Scenario: Bind service 1 to app 1
+   ...
+
+   Scenario: Bind service 1 to app 2
+   ...
+```
+
+Now, when you want to execute just that scenario out of the whole suite, use the `TEST_ACCEPTANCE_TAGS` environmental variable to set the tag.
+
+To run single scenario tagged with `@wip`:
+
+```bash
+TEST_ACCEPTANCE_TAGS=@wip make test-acceptance
+.
+.
+.
+
+```
+
+Only the scenario or feature tagged with the particular tag (`@wip` in case of our example) will be executed, and all of the others will be simply skipped.
+
+```bash
+.
+.
+.
+
+1 feature passed, 0 failed, 3 skipped
+1 scenario passed, 0 failed, 16 skipped
+10 steps passed, 0 failed, 179 skipped, 0 undefined
+Took 0m49.231s
+```
+
+## Inspect test results
+
+During the test execution the result of individual steps are shown as the execution goes. That is the most obvious way to see the results, but it is far from elegant - especially with a growing number of scenarios. However, the framework also records the test results in a standardized way of xUnit format, which can be found under `$WORKSPACE/out/acceptance-tests` directory.
```

Comment:
Should introduce somewhere `test-acceptance-artifacts` rule and its usage? should we really call it explicitly or that could a part of test-acceptance rule execution?

Reply:
The `test-acceptance-artifacts` is a separated target [used in CI](https://github.com/openshift/release/blob/3a0faadab90334bc3295700207f30f44f0f3359c/ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master__4.5.yaml#L31) to collect artifacts from acceptance tests to be archived. It is a separated target because we want to ensure the artifacts are collected no matter how the actual acceptance test execution goes - We wouldn't want to loose the artifacts if the `test-acceptance` fails.